src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -646,28 +646,28 @@ def add_error_to_status(self, message, error_code=Constants.PatchOperationErrorC
         current_operation = self.__current_operation if current_operation_override_for_error == Constants.DEFAULT_UNSPECIFIED_VALUE else current_operation_override_for_error
 
         if current_operation == Constants.ASSESSMENT:
-            self.__add_error(self.__assessment_errors, error_detail)
-            self.__assessment_total_error_count += 1
-            # retain previously set status and code for assessment substatus
-            if self.__assessment_substatus_json is not None:
-                self.set_assessment_substatus_json(status=self.__assessment_substatus_json["status"], code=self.__assessment_substatus_json["code"])
-            else:
-                self.set_assessment_substatus_json()
+            if self.__add_error(self.__assessment_errors, error_detail):
```

Comment:
Rename the __add_error method to __try_add_error to make it immediately obvious that there are circumstances where the message add request won't be honored.

Reply:
Thanks, done